client/src/Carousel.jsx

```diff
@@ -1,17 +1,32 @@
 import React from 'react';
 import T from 'prop-types';
 import ENV from './config';
+import Item from './Item';
 
-const Carousel = ({ items, position, onItemClick }) => (
-  <div data-position={position}>
-    {items.map((item) => <button type="button" key={item.id} data-id={item.id} onClick={onItemClick}>{item.name}</button>)}
-  </div>
+const Carousel = ({
+  items, position, onItemClick, title,
+}) => (
+  <>
+    <div className="g-carousel-title">{title}</div>
+    <div data-position={position} className="g-carousel-container">
+      <div className="g-carousel-arrow left" data-dir="-1">
+        <img src="./res/arrow-left.svg" alt="Left Arrow" className="g-carousel-arrow-image" />
+      </div>
+      <div className="g-item-slider">
+        {items.map((item) => <Item key={item.id} item={item} onItemClick={onItemClick} />)}
```

Comment:
Has the linter given you any issues with using just a number for the key? I had a problem with it yesterday, but that may have solely been due to it being an index from the mapping function.